tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/TeamMailboxMemberGetMethodContract.scala

```diff
@@ -0,0 +1,310 @@
+import org.apache.james.jmap.core.ResponseObject.SESSION_STATE
+import org.apache.james.jmap.http.UserCredential
+import org.apache.james.jmap.rfc8621.contract.Fixture.{ACCEPT_RFC8621_VERSION_HEADER, ANDRE, ANDRE_PASSWORD, BOB, BOB_PASSWORD, DOMAIN, authScheme, baseRequestSpecBuilder}
+import org.apache.james.utils.DataProbeImpl
+import org.junit.jupiter.api.{BeforeEach, Test}
+
+trait TeamMailboxMemberGetMethodContract {
+  @BeforeEach
+  def setUp(server: GuiceJamesServer): Unit = {
+    server.getProbe(classOf[DataProbeImpl])
+      .fluent()
+      .addDomain(DOMAIN.asString)
+      .addUser(BOB.asString(), BOB_PASSWORD)
+      .addUser(ANDRE.asString(), ANDRE_PASSWORD)
+
+    requestSpecification = baseRequestSpecBuilder(server)
+      .setAuth(authScheme(UserCredential(BOB, BOB_PASSWORD)))
+      .addHeader(ACCEPT.toString, ACCEPT_RFC8621_VERSION_HEADER)
+      .build()
+  }
+
+  @Test
+  def missingPublicAssetCapabilityShouldFail(): Unit =
+    `given`
+      .body(
+        s"""{
+           |	"using": ["urn:ietf:params:jmap:core"],
+           |	"methodCalls": [
+           |		[
+           |			"TeamMailboxMember/get",
+           |			{
+           |				"accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+           |				"ids": null
+           |			},
+           |			"c1"
+           |		]
+           |	]
+           |}""".stripMargin)
+    .when
+      .post
+    .`then`
+      .statusCode(SC_OK)
+      .contentType(JSON)
+      .body("", jsonEquals(
+        s"""{
+           |	"sessionState": "${SESSION_STATE.value}",
+           |	"methodResponses": [
+           |		[
+           |			"error",
+           |			{
+           |				"type": "unknownMethod",
+           |				"description": "Missing capability(ies): com:linagora:params:jmap:team:mailboxes"
+           |			},
+           |			"c1"
+           |		]
+           |	]
+           |}""".stripMargin))
+
+
+  @Test
+  def shouldFailWhenWrongAccountId(): Unit =
+    `given`
+      .body(
+        s"""{
+           |	"using": ["urn:ietf:params:jmap:core", "com:linagora:params:jmap:team:mailboxes"],
+           |	"methodCalls": [
+           |		[
+           |			"TeamMailboxMember/get",
+           |			{
+           |				"accountId": "unknownAccountId",
+           |				"ids": null
+           |			},
+           |			"c1"
+           |		]
+           |	]
+           |}""".stripMargin)
+      .when
+      .post
+      .`then`
+      .statusCode(SC_OK)
+      .contentType(JSON)
+      .body("", jsonEquals(
+        s"""{
+           |  "sessionState": "${SESSION_STATE.value}",
+           |  "methodResponses": [
+           |    ["error", {
+           |      "type": "accountNotFound"
+           |    }, "c1"]
+           |  ]
+           |}""".stripMargin))
+
+  @Test
+  def getShouldReturnEmptyListByDefault(): Unit =
+    `given`
+      .body(
+        s"""{
+           |	"using": ["urn:ietf:params:jmap:core", "com:linagora:params:jmap:team:mailboxes"],
```

Comment:
missing the `urn:ietf:params:jmap:mail` capability

Reply:
Ah you are right actually, we use it for revoke access method as well